t/page/missing_country.rb

```diff
@@ -0,0 +1,15 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/missing_country'
+require 'pry'
+
+describe 'MissingCountry' do
+  it 'a missing country has a display name' do
```

Comment:
a fairly tiny nit, but if we're going to use 'it' style, then it's good if it actually makes sense as a sentence. The context is already a Missing Country, so I think it's OK to just say "it 'has a display name'"